README.md

```diff
@@ -16,11 +16,12 @@ educators to increase student engagement and make learning fun.
 ### System Requirements
 
 1. Ruby (>= 2.3.1)
-2. Ruby on Rails
-3. PostgreSQL
-4. ImageMagick or GraphicsMagick (For [MiniMagick](https://github.com/minimagick/minimagick))
-5. Node.js
-6. Yarn
+1. Ruby on Rails
+1. PostgreSQL
+1. ImageMagick or GraphicsMagick (For [MiniMagick](https://github.com/minimagick/minimagick))
+1. Ghostscript (for PDF to image processing for [MiniMagick](https://github.com/minimagick/minimagick))
```

Comment:
Imagemagick deals only with images. However they have 'drivers' or 'sub-libraries' that helps do cross file-type conversion. Ghostscript helps to provide imagemagick functionality for pdf, so we need it for the scribing question. 

This took me a while to debug on my local machine cause imagemagick didn't have error messages for this >_< 

Surprisingly travis has this already, but we will need to ensure that our staging and production dependencies are updated for this as well. 

cc: @fonglh